docker-runtime-hook/entrypoint.sh

```diff
@@ -1,11 +1,10 @@
 #/bin/bash
```

Comment:
are these changes relevant?

Reply:
It's a cosmetic change that somehow fell through the cracks. As I am the original author of this file and feel a bit guilty about that, I wanted to fix it. Can I, without a separate PR, pleeeeaaaase? :P